### ISSUE TITLE
无限卡 macOS Installer 重启

### ISSUE THREAD

**Issue by zhangz1han** [jingkunchen/Chromebox3-Sonoma-Hackintosh#3]
镜像来自 [MEDIY](https://macos.mediy.cn/macOS%20Sonoma%2014/)

尝试了 [macOS Sonoma 14.6.1 (23G93)](https://macos.mediy.cn/macOS%20Sonoma%2014/macOS%20Sonoma%2014.6.1%20(23G93)/)，刻录时报错，安装时报损坏，多次重新下载依旧，怀疑源文件有损坏

然后尝试 [macOS Sonoma 14.5 (23F79)](https://macos.mediy.cn/macOS%20Sonoma%2014/macOS%20Sonoma%2014.5%20(23F79)/)，刻录成功，分区、安装成功后重启。然后在 OpenCore 中启动 macOS Installer，打印日志后屏幕花屏变蓝色卡一会儿，成功出现苹果 Logo 和下方剩余 XX 分钟进度条。读条完成后再次重启，但 OpenCore 中还是 macOS Installer，启动后打印大量日志然后自动重启，OpenCore 中还是 macOS Installer，陷入无限死循环

之前唯一一次成功是用国光酱的第一版 EFI（第二版会花屏卡死）安装 macOS 13，因需要用 Xcode 16.1 不得不升级 14.5+

BIOS 版本为 2408.1 09/14/2024

网卡：Intel AX200
固态：西数 SN720 512G（分区为 APFS 区分大小写）
内存：铨兴 16G×2（实际识别到共16G）
启动盘：闪迪 CZ600 64G
刻录：balenaEtcher 1.19.25
EFI 修改：DiskGenius 5.6.1

排查不出问题，求问有没有遇到相同问题的朋友 QAQ

**Comment by jingkunchen** [comment]
安装和运行不是同一个efi,要屏蔽网卡等一些东西

**Comment by khgaurav** [comment]
您好！我也遇到了相同的问题。很抱歉我不会中文，但我真的很需要您的帮助。
请问您是否已经解决了这个问题？如果已经解决，能否分享一下解决的具体步骤？
我也是在安装 macOS Sonoma 时遇到类似情况：
14.5 版本安装后进入循环重启
OpenCore 启动时显示花屏
如果您已找到解决方案，请告诉我详细的操作步骤，感激不尽！
再次为我不会中文表示歉意。
谢谢！

**Comment by zhangz1han** [comment]
> 您好！我也遇到了相同的问题。很抱歉我不会中文，但我真的很需要您的帮助。 请问您是否已经解决了这个问题？如果已经解决，能否分享一下解决的具体步骤？ 我也是在安装 macOS Sonoma 时遇到类似情况： 14.5 版本安装后进入循环重启 OpenCore 启动时显示花屏 如果您已找到解决方案，请告诉我详细的操作步骤，感激不尽！ 再次为我不会中文表示歉意。 谢谢！

No, the problem still exists and I haven't found a solution yet

**Comment by khgaurav** [comment]
Were you able to get any other version of macos working? Please share your EFI folder if you were able to. Thank you!

**Comment by zhangz1han** [comment]
> Were you able to get any other version of macos working? Please share your EFI folder if you were able to. Thank you!

I successfully booted macOS 13 using [this EFI](https://github.com/sqlsec/Asus-ChromeBox-i7-8550U)

**Comment by khgaurav** [comment]
I was able to boot into macOS 14. Just had to use a different config.plist when the green screen appears. Have uploaded it in my forked repo https://github.com/khgaurav/ASUS-Chromebox3-OpenCore/tree/main.
